sdks/java/expansion-service/build.gradle

```diff
@@ -38,6 +38,7 @@ dependencies {
   compile project(path: ":sdks:java:core", configuration: "shadow")
   compile project(path: ":runners:core-construction-java")
   compile project(path: ":runners:java-fn-execution")
+  compile project(path: ":runners:direct-java")
```

Comment:
This is unfortunate, but PipelineOptionsFactory does not permit not to have a runner.